### PR TITLE
Address some styling feedback

### PR DESCRIPTION
**Pull request by cbelan-d2l**
Still more to do

This is just adding in a bit more padding.
Until I get the updated styles from design

image

### CODE REVIEW

**Comment by IhabAlkabary** [comment]
Can you share a printscreen?

**Comment by cbelan-d2l** [comment]
> Can you share a printscreen?

Updated in description

File: src/components/leaderboard-row.js [hunk 1]
```diff
@@ -40,8 +40,8 @@ class LeaderboardRow extends BaseMixin(LitElement) {
 				border-radius: 15px;
 				background-color: #E3E9F1;
 				border: 1px solid #E3E9F1;
-				height: 15px;
-				width: 15px;
+				height: 21px;
```
Comment (IhabAlkabary) [inline src/components/leaderboard-row.js]: Should be using pixels, or shall we use em or rem.
Maybe a quick question in the ux-dev channel or to Margaree?

Comment (cbelan-d2l) [reply src/components/leaderboard-row.js]: This is basically just a quick hack thing to get it to look a bit better until I have the actual designs.
Good point that em/rem will make it better for responsiveness
Would be acceptable for now and will fix in a future PR?

Comment (IhabAlkabary) [reply src/components/leaderboard-row.js]: Follow-up PR is fine since this is not finalized.

Comment (cbelan-d2l) [reply src/components/leaderboard-row.js]: Thanks!

**Pull request merged**